### PR TITLE
issue 1468: add queue annotation in deployment example and add queue.…

### PR DESCRIPTION
**Pull request by nolimitkun**
…yaml

### CODE REVIEW

**Review by Thor-wl — COMMENTED** [review]
Please add --signoff when push the commit

**Comment by nolimitkun** [comment]
> Please add --signoff when push the commit

ok,will keep in mind for next time.

File: example/deployment/deployment.yaml [hunk 1]
```diff
@@ -13,8 +13,12 @@ spec:
     metadata:
       labels:
         app: nginx
+      annotations:
+        #creat "test queue and use this annotation
```
Comment (shinytang6) [inline example/deployment/deployment.yaml:17]: creat => create
rest /lgtm

Comment (nolimitkun) [reply example/deployment/deployment.yaml]: typo fixed

**Comment by nolimitkun** [comment]
please review.

File: example/deployment/deployment.yaml [hunk 2]
```diff
@@ -13,8 +13,12 @@ spec:
     metadata:
       labels:
         app: nginx
+      annotations:
+        #create "test queue and use this annotation
```
Comment (shinytang6) [inline example/deployment/deployment.yaml:17]: Please remove the quotations

Comment (nolimitkun) [reply example/deployment/deployment.yaml]: yep, done

**Review by shinytang6 — APPROVED** [review]
Thanks.
/lgtm
/approve

**Comment by Thor-wl** [comment]
This PR will be merged for it just add some examples and the CI doesn't work well.

**Pull request merged**